src/features/dataInfo.js

```diff
@@ -0,0 +1,18 @@
+ export const dataInfo = (user, time) => {
+    const dateCreated = new Date(time*1000);
+    const dateCurrent = new Date();
+    const minutesDiff = Math.floor(Math.abs((dateCurrent - dateCreated) / (1000 * 60)));
+    const hoursDiff = Math.floor(minutesDiff / 60);
+    const daysDiff = Math.floor(hoursDiff / 24);
+
+    if(minutesDiff < 60) {
+        return `Posted by ${user}, ${minutesDiff} minutes ago`
```

Comment:
I would remove user from this function. Keep it simple for just the time ago calculation